pkgs/top-level/python-packages.nix

```diff
@@ -8461,6 +8461,19 @@ in {
     };
   };
 
+  ps_mem = buildPythonPackage rec {
```

Comment:
it does not look like a library too me. Can you move it out of `python-packages.nix` into its own directory (ex: `./pkgs/tools/system`) and reference it in `all-packages.nix`.